packages/extension-ui/src/Popup/CreateAccount/SafetyFirst.tsx

```diff
@@ -22,6 +22,24 @@ function SafetyFirst({ className, onNextStep }: Props): React.ReactElement<Props
 
   const goTo = useCallback((path: string) => () => onAction(path), [onAction]);
 
+  const footer = (
+    <HelperFooter>
+      <img
+        className='icon'
+        src={helpIcon}
+      />
+      <span>
+        {t<string>('Why it is critical to store your secret phrase in a safe place?')}
+        <span
+          className='link'
+          onClick={goTo('/help-safety')}
+        >
+          {` ${t<string>('Learn more')}`}
```

Comment:
Cool idea. I would also use `&nbsp;`where we need extra empty space rather than leaving the space itself as it's easy to miss in a string template.
```js
{`${t<string>('Why it is critical to store your secret phrase in a safe place?')}`}&nbsp;
```